include/nbl/video/IAPIConnection.h

```diff
@@ -14,6 +14,11 @@ class IPhysicalDevice;
 class IAPIConnection : public core::IReferenceCounted
 {
 public:
+    enum E_EXTENSION
+    {
+        E_SURFACE = 0
+    };
```

Comment:
call them features not extensions (cause once Extension gets promoted to core in Vulkan it becomes a Feature)

Reply:
resolved